estela-web/src/utils.ts

```diff
@@ -40,63 +39,54 @@ export function setValArr({ arr, val, index }: { arr: number[]; val: number; ind
 
 export function parseDuration(duration: string | undefined): Duration {
     if (duration && duration !== "undefined") {
-        const durationRegex = /(?:(\d+) days?,\s*)?(\d{1,2}):(\d{1,2}):(\d{1,2})(?:\.(\d+))?/;
+        const durationRegex = /(?:(\d+) days?,\s*)?(\d{1,2}):(\d{1,2}):(\d{1,2}?\.(\d+))?/;
```

Comment:
Test fails when I put the following string: "00:10:23".
Output: {"days":0,"hours":0,"minutes":10,"seconds":null}

Reply:
Please re-review

Reply:
It's OK now